halpybot/packages/models/seal.py

```diff
@@ -0,0 +1,31 @@
+"""
+seal.py - Who are you, again?
+
+Copyright (c) The Hull Seals,
+All rights reserved.
+
+Licensed under the GNU General Public License
+See license.md
+"""
+from typing import Optional, List
+from attr import dataclass
+
+
+@dataclass
```

Comment:
inconsistent usage of `define` and `dataclass` across this PR.
I recommend using `@define` for everything.